packages/paypal-express-integration/src/paypal-express-button-initialize-options.ts

```diff
@@ -0,0 +1,48 @@
+import {
+    PaypalStyleOptions
+} from './paypal-express-types';
+import { StandardError } from '@bigcommerce/checkout-sdk/payment-integration-api';
+
+export interface WithPaypalExpressButtonInitializeOptions {
+    paypal: PaypalExpressButtonInitializeOptions;
+}
```

Comment:
May I ask you to place this interface below the PaypalExpressButtonInitializeOptions?

Reply:
Fixed